src/components/network/network-map.js

```diff
@@ -252,8 +252,9 @@ const NetworkMap = (props) => {
 
     function renderTooltip() {
         return (
-            tooltip &&
-            tooltip.visible && (
+            tooltip?.visible &&
```

Comment:
suggestion : 
    const shouldRenderTooltip =
        tooltip?.visible &&
        !props.mapEquipments?.getHvdcLine(tooltip.equipmentId);

Reply:
Thanks for the suggestion, but I won't do a refactorization in this US.